templates/patterns/link/link.ui_patterns.yml

```
@@ -28,9 +34,11 @@ link:
     remove_icon_spacers:
       type: boolean
       label: Remove icon spacers
+      description: When enabled, not add the margin class to the icon
     link_id:
       type: textfield
       label: Button id
+      description: Add id attribute to the link.
```

```suggestion
      description: A specific ID to use for the link.
```
Not sure this should be left at all.